sass/components/_status-icon.scss

```diff
@@ -193,3 +193,18 @@
         top: -1px;
     }
 }
+
+.status-dropdown-menu {
+    .online--icon > svg {
+        fill: rgba(var(--online-indicator-rgb), 1);
+    }
+    .away--icon > svg {
+        fill: rgba(var(--away-indicator-rgb), 1);
+    }
+    .dnd--icon > svg {
+        fill: rgba(var(--dnd-indicator-rgb), 1);
+    }
+    .offline--icon {
+        fill: rgba(var(--center-channel-color-rgb), 0.6);
```

Comment:
I believe we have variables for the different opacities of the center channel (in increments of 8%). So for this I think we should be able to use:

`fill: var(--center-channel-color-56);`

In fact, I'm not sure we need the `rgba()` wrapping the variables here at all.

Reply:
Sure thing, I'll adjust the alpha to `0.56`.

Apps team had a thread with @asaadmahmood about the variables and the `rgba()` function. The plan moving forward is to implement using the above pattern, but follow the guidelines set forth by design in regards to the `alpha` values.

Asaad provided the following for our reference.
<img width="999" alt="Screen Shot 2020-06-01 at 9 26 58 AM" src="https://user-images.githubusercontent.com/44858354/83430192-0fbde780-a3ea-11ea-81fd-77d84d1381f9.png">

And in regards to the `rgba()` function, it's a CSS function that accepts an "rgb triplet" like '255, 255, 255'. I've added a `-rgb` value for each of the theme global variables. These will be used to apply opacity changes in the `.scss` rather than the `changeOpacity` function in the javascript.